### ISSUE TITLE
Dataset: Flow and Boundary Condition

### ISSUE THREAD

**Issue by a-parida12** [loliverhennigh/Steady-State-Flow-With-Neural-Nets#2]
The train.tfrecords I think does not have the flow conditions stored in the boundarys and only has the geometry info.  Am I Missing out on something on how to add flow condition to the network?
![image](https://user-images.githubusercontent.com/23334003/43315885-be0445bc-9197-11e8-81ec-32c173e4c9f6.png)

**Comment by loliverhennigh** [comment]
That is correct, the flow conditions are not input to the network but they are kept consistent through the dataset. I have since played around with adding flow conditions with some success.
